src/test/ui/deref-suggestion.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0308]: mismatched types
+  --> $DIR/deref-suggestion.rs:14:9
+   |
+14 |     foo(&"aaa".to_owned());
+   |         ^^^^^^^^^^^^^^^^^ expected struct `std::string::String`, found reference
+   |
+   = note: expected type `std::string::String`
+              found type `&std::string::String`
+   = help: try with `*&"aaa".to_owned()`
```

Comment:
@durka: It's pretty simple so I'll do it.
@kennytm: Hum... I guess I'll need to add a check over the dereference...